### ISSUE TITLE
Feature Request: Minimum bar size config option

### ISSUE THREAD

**Issue by PaulWieland** [fanthos/chartjs-chart-timeline#4]
It would be nice if there was an option for this chart type to specify the minimum bar size in pixels. I have a 24 hour timeline and events that only last a minute or two do not show up at all.

The use case I have is to chart all of the events that occur in my house (motion detectors, doors open/close, lights on/off, etc) but for events that only last a few minutes, the bars render to small to even know they are there or to mouse over them.

**Comment by fanthos** [comment]
Do you mean the bar width?
You can use the resize feature from chartjs to resize the chart.

**Comment by PaulWieland** [comment]
Yes the minimum bar width. Imagine what a 30 second event looks like on a 24 hour time scale (you can't see it at all). Would like to be able to force it to 10px or 20px so that its visible and also so you can hover over it to see its tooltip.

**Comment by fanthos** [comment]
Got it. This feature may added in future but currently no plan. You can change the code `var width = end - x;` to `var width = Math.max(...)` in updateElement function to workaround this issue.
